python/lsst/sattle/sattlePy.py

```diff
@@ -299,7 +327,7 @@ def satellite_tracks(self, psf, sat_coords):
                 print(corner1[0][i], corner2[1][i], corner2[0][i], corner2[1][i], corner3[0][i],
                       corner2[1][i], corner4[0][i], corner4[1][i])
         # TODO: This is for testing only, remove
```

Comment:
Please either remove the comment or remove the thing

Reply:
Keeping for now as it will be removed when unit tests are complete.